src/simulate.jl

```diff
@@ -81,7 +83,7 @@ function simulate(p, biomass; start::Int64=0, stop::Int64=500, use::Symbol=:nons
       alg = Tsit5()
   end
 
-  sol = solve(prob, alg, callback = cb, saveat=t_keep, dense=false, save_timeseries=false)
+  sol = solve(prob, alg, callback = cb, dtmax = 1, saveat=t_keep, dense=false, save_timeseries=false)
```

Comment:
Curious that this was found necessary. Did you try taking this off after the other checks were put on? Otherwise, if you'd like to give a test problem which needs it to me, I'd be happy to investigate what's going on. These floating point issues can be a real PITA.